protocol/xmrtaker/instance_test.go

```diff
@@ -3,17 +3,22 @@ package xmrtaker
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
-	"github.com/athanorlabs/atomic-swap/monero"
+	"github.com/athanorlabs/atomic-swap/common/types"
 )
 
-func TestGetAddress(t *testing.T) {
-	c := monero.CreateWalletClient(t)
-	addr, err := getAddress(c, "", "")
+func TestNewInstance(t *testing.T) {
```

Comment:
I removed the `getAddress` function and `TestGetAddress` was the only unit test for `instance.go`. To keep the file and get a little coverage, I added `TestNewInstance`.